src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started working");

```

Comment:
The message 'Bulldozer started working' is not very informative. Consider specifying what exactly the Bulldozer is doing, for example, 'Bulldozer is moving earth.'